docs/tutorials/viz_listBox.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""
+=========
+ListBox
+=========
+
+This example shows how to use the UI API. We will create a list
+some geometric shapes from DIPY UI elements.
+
+First, a bunch of imports.
+"""
+from fury import ui, window
+
+###############################################################################
+# Create some text blocks that will be showm when
+# list elements will be selected
+
+Get_text = ui.TextBlock2D(text="Get", font_size=30, position=(500, 400))
```

Comment:
`Get` word can be confusing.  replace it with `welcome_text` and `text="Say welcome"`